trunk/source/BA_SiteRepository/Ultimate.target

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<target name="Ultimate E4.37">
+    <targetJRE path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-21"/>
+    <launcherArgs>
+        <vmArgs>-ea -Xms4M -Xmx4G</vmArgs>
+    </launcherArgs>
+    <locations>
+        <location includeAllPlatforms="true" includeConfigurePhase="true" includeMode="planner" includeSource="false" type="InstallableUnit">
+            <repository id="eclipse-releases" location="https://download.eclipse.org/releases/2025-09"/>
+            <unit id="org.eclipse.cdt.feature.group" version="0.0.0"/>
+            <unit id="org.eclipse.cdt.sdk.feature.group" version="0.0.0"/>
+            <unit id="org.eclipse.equinox.sdk.feature.group" version="0.0.0"/>
+            <unit id="org.eclipse.sdk.feature.group" version="0.0.0"/>
+            <unit id="org.eclipse.jdt.feature.group" version="0.0.0"/>
+            <unit id="org.eclipse.equinox.server.jetty.feature.group" version="0.0.0"/>
+        </location>
+        <location includeAllPlatforms="true" includeConfigurePhase="true" includeMode="planner" includeSource="false" type="InstallableUnit">
+            <repository id="orbit-aggregation" location="https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/2025-09"/>
+            <unit id="org.eclipse.orbit.legacy.feature.group" version="0.0.0"/>
```

Comment:
You replaced all the versions by 0.0.0. Does this mean that any version that is found in the repo is accepted and thus makes future framework updates even simpler?